docs/getting-started.md

```diff
@@ -1,124 +1,119 @@
-
 # Getting Started with Metabase
 
 Metabase is a simple and powerful analytics tool which lets anyone learn and make decisions from their company's data. No technical knowledge required! We hope you love it.
 
 ## Logging in
+
 The way you log in to Metabase will depend on how you or your admin set it up, so if you don’t know where to go, just ask the person who sent you your Metabase invite.
 
 ## The homepage
-Go ahead and log in to Metabase and the first thing you'll see is the activity feed. Right now there’s not much there, but it’ll soon get full as you and your teammates do things in Metabase.
 
-![Activity Feed](images/ActivityFeed.png)
+![The home page](./users-guide/images/metabase-homepage.png)
+
+Fresh out of the box, Metabase will show you a few things on the home page:
+
+- Some [automatic explorations](./users-guide/14-x-rays.md) of your tables that you can look at and save as a dashboard if you like any of them.
+- An area where things you or your teammates create will show up, along with a link to see all the dashboards, questions, and pulses you have.
+- A list of the databases you've connected to Metabase.
+
+![Our data](./users-guide/images/our-data.png)
 
 ## Asking a new question
 
 But, enough about that — let’s get to asking questions. For the next few examples, we'll be using the **Sample Dataset** that comes with Metabase.
 
-Go ahead and click **New Question** at the top of the screen. Now we’re on the new question page. The bar that you see going across the page is what you’ll use to ask your questions. The “Select a table” dropdown should already be open, showing you a list of your database(s) and the tables within them.
+Go ahead and click **Ask a question** at the top of the screen. There are several ways you can ask a question in Metabase, but we'll click on the Simple Question option for now. You'll then need to pick the table that you have a question about. Once you do, you'll see the table's contents.
 
-![Query Builder](images/QueryBuilder.png)
+![The Orders table](images/OrdersTable.png)
 
 ## Our first question
 
-We’re going to ask a question about the Orders table in the Sample Dataset database, so we’ll click on **Orders** in the dropdown. The Orders table has a bunch of fake data in it about product orders for a made up company. If you just want to see everything that’s in this table, you can click **Run query** and see all the raw data.
+The Orders table has a bunch of fake data in it about product orders for a made up company. Let's start with a simple question about these orders: how many orders have been placed with a subtotal (before tax) greater than \$40? More precisely, this question translates to, "How many records (or rows) are in the table 'Orders' with a value greater than 40 in the Subtotal column?”
```

Comment:
Super picky, but I think it should be "translates to:" instead of "translates to,"